packages/tests-e2e/.env.example

```diff
@@ -3,6 +3,6 @@ PLAYWRIGHT_TEST_URL=http://localhost:3000
 CORE_API_URL=https://app.staging1.cbhosts.com
 BACKEND_API_URL=https://api.staging1.cbhosts.com
 FRONTEND_API_URL_SUFFIX=frontendapi.staging1.cbhosts.com
-PLAYWRIGHT_JWT_TOKEN=
-PLAYWRIGHT_MICROSOFT_EMAIL=
-PLAYWRIGHT_MICROSOFT_PASSWORD=
+PLAYWRIGHT_JWT_TOKEN=eyJhbGciOiJSUzI1NiIsImtpZCI6InBraS03OTk4NzU5MTQxNTk5ODY5Mzk4IiwidHlwIjoiSldUIn0.eyJpc3MiOiJhdXRoLnN0YWdpbmcxLmNiaG9zdHMuY29tIiwic3ViIjoidXNyLTIyMjA0MTIzOTEyOTkzMjI5NzEiLCJleHAiOjE3MTcwNTg0NzgsIm5iZiI6MTcxNDA1ODQ2OCwiaWF0IjoxNzE0MDU4NDc4LCJuYW1lIjoic3lzdGVtdGVzdCIsIm9yaWciOiJhbmRlcnMuY2hvaStzeXN0ZW10ZXN0QGNvcmJhZG8uY29tIiwiZW1haWwiOiJhbmRlcnMuY2hvaStzeXN0ZW10ZXN0QGNvcmJhZG8uY29tIiwidmVyc2lvbiI6Mn0.GmEafXbSSVlq27q3elg-aOhUqNTwBjUISYh9Hmf2ryml27etrHuoPOtMI43LXaCiVncjec_y_RaOEBHmJEZPGWhE4Czc1poppWxzPxFo0RaXPQp4QHgqcPE_ApCpOZXgdfVDIpNl3sNniayxqEalCa1yxN5xVgqx4fccPrjGFrDslQekqqgby5bz3MgG_HXVxsVKP1PomzrRc-Rw13jxhO9YHsWPbCoHBnMnXHQ45I8_a_zxWntlGbg2ojzvNNyME4mFOCMTajF-m_LkIJAQ-rYR6D2ohFXhJyhfyTJRjASb8Gd080s0AbmSpBbJkWEWoNDX8i6t9AJ_aIIwlih33Q
+PLAYWRIGHT_MICROSOFT_EMAIL=corbado.systemtest@outlook.com
```

Comment:
we must not check in those credentials as this repository is public => please remove them and @aehnh we should change the password now

Reply:
removing them in next commit